aws-android-sdk-mobile-client/src/androidTest/java/com/amazonaws/mobile/client/AWSMobileClientOfflineTest.java

```diff
@@ -113,6 +115,7 @@ public void testGetConfiguration() throws JSONException {
         }
     }
 
+    @Ignore("This test may get stuck")
```

Comment:
This needs to be documented that the network conditions might be switched during test run causing the test to fail.

Reply:
done